### PR TITLE
JIRA: QB-2111 fork react avartar and fix the node engine issue

### PR DESCRIPTION
**Pull request by Bunchhieng**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,19 +14,19 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/sitebase/react-avatar.git"
+    "url": "https://github.com/QuickBase/react-avatar.git"
   },
   "author": "Wim Mostmans <wim@sitebase.be>",
```
Comment (Kunjan) [inline package.json]: what ramification is there with keeping the same "author" and retaining the same MIT license? we may want to double check with Chris P about the correct process of doing this

File: package.json [hunk 2]
```diff
@@ -17,19 +14,19 @@
...
   "bugs": {
-    "url": "https://github.com/sitebase/react-avatar/issues"
+    "url": "https://github.com/QuickBase/react-avatar/issues"
```
Comment (nvanselow) [inline package.json]: I vote we leave bugs and homepage as the original. We are only a fork. We are not planning on supporting this any longer than we need to.

File: package.json [hunk 3]
```diff
@@ -3,9 +3,6 @@
   "version": "2.5.1",
```
Comment (nvanselow) [inline package.json]: The name needs to be "@quickbase/react-avatar" to work with nexus.

**Comment by Bunchhieng** [comment]
Updated. @nvanselow @Kunjan

**Review by nvanselow — APPROVED** [review]
I think @Kunjan makes a good point about running this by @cpliakas, but in the meantime let's go forward with this. I'll stop by so we can deploy this to nexus. We can always change it if we  get different direction later.

**Pull request merged**